src/Microsoft.OData.Core/ODataMessageReaderSettings.cs

```diff
@@ -194,6 +194,7 @@ public ODataMessageQuotas MessageQuotas
         /// <summary>
         /// Whether to read untyped values as a raw string.
         /// </summary>
+        [Obsolete("This will be dropped in the 9.x release. Please use 'PrimitiveTypeResolver' for unknown types.")]

```

Comment:
```suggestion
        [Obsolete("ReadUntypedAsString is deprecated and will be dropped in a future release. Please use 'PrimitiveTypeResolver' for primitive unknown types.")]
```

Reply:
Thanks. I changed using your sentence.

By the way, I am using the 'same' sentence' following up the existing 'Obsolete'. @challenh Can we consider changing others?

Reply:
see: https://github.com/OData/odata.net/blob/dev-8.x/src/Microsoft.OData.Core/Json/IJsonWriter.cs#L26, etc.